test/e2e/data/kustomize/default/kustomization.yaml

```diff
@@ -1,8 +1,21 @@
 ---
 resources:
 - ../../../../../kustomize/v1alpha7/default
+- ccm.yaml
 
 components:
 - ../common-patches/cni
 - ../common-patches/containerd
-- ../upgrade-patches
```

Comment:
@lentzi90 Can you check this out? It doesn't feel right to remove this.

Reply:
Yes this is needed for the clusterctl upgrade test (part of the e2e-full suite). I guess what happened here was that we only use the default template in those tests, so the external-cloud-provider kustomizations didn't have the upgrade-patches and that is why it was removed.
Please add it back and the e2e-full test should hopefully pass :slightly_smiling_face: